review_practice/react-app/src/containers/SearchBar/ModalBody/PriceFilter.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from 'react';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+import PriceRangeSlider from 'components/PriceRangeSlider';
+
+const MIN_PRICE = 0;
+const MAX_PRICE = 100000;
+
+export default ({ buttonState, setButtonState }) => {
+    // Context
+    const {
+        searchOption: {
+            searchOptions: { price },
+        },
+        dispatchSearchOption,
+    } = useContext(SearchRoomContext);
+
+    // State
+    const priceState = useState([
+        price && price.startPrice ? price.startPrice : MIN_PRICE,
+        price && price.endPrice ? price.endPrice : MAX_PRICE,
+    ]);
+
+    // Handler
+    const changeButtonName = price => {
+        const [startPrice, endPrice] = price;
+        setButtonState({ ...buttonState, buttonName: `￦${startPrice} - ￦${endPrice}` });
+    };
+
+    const reflectPriceFilter = price => {
+        dispatchSearchOption({
+            type: 'setPriceFilter',
+            payload: { startPrice: price[0], endPrice: price[1] },
+        });
+        setButtonState({ ...buttonState, isClicked: false });
+    };
```

Comment:
```suggestion
    const reflectPriceFilter = ([ startPrice, endPrice ])=> {
        dispatchSearchOption({
            type: 'setPriceFilter',
            payload: { startPrice, endPrice },
        });
        setButtonState({ ...buttonState, isClicked: false });
    };
```
이렇게 명시적으로 표현해주면 더 좋을 것 같아요!